.travis.yml

```diff
@@ -4,11 +4,6 @@ services:
 
 matrix:
     include:
-        - os: linux
```

Comment:
Can you add LLVM-10?

Reply:
`docker build --build-arg BASE_IMAGE=ubuntu:19.10 --build-arg IMAGE_REPO=eoan --build-arg TARGET_LLVM_VERSION=10 .`

should be the invocation

Reply:
We shouldn't block this PR on adding llvm-10 right?

Reply:
Sure, this can come through another PR.